web/src/SidebarAccount.tsx

```diff
@@ -0,0 +1,268 @@
+  font-family: ${Font.monospace};
+  font-size: ${FontSize.small};
+  color: ${Color.grayLightest};
+  margin-right: ${SizeUnit(0.5)};
+  opacity: 0;
+  transition: opacity ${AnimDuration.short} ease;
+  transition-delay: ${AnimDuration.default};
+`
+
+let SidebarAccountButton = styled.button`
+  display: flex;
+  align-items: center;
+  border: 0;
+  cursor: pointer;
+  background-color: transparent;
+  padding-left: ${SizeUnit(0.25)};
+  padding-right: ${SizeUnit(0.25)};
+  padding-top: ${SizeUnit(0.15)};
+  padding-bottom: ${SizeUnit(0.15)};
+
+  &:hover ${SidebarAccountLabel} {
+    opacity: 1;
+  }
+`
+let SidebarAccountIcon = styled(AccountIcon)`
+  fill: ${Color.blue};
+  transition: color ${AnimDuration.default} linear;
+
+  &:hover {
+    fill: ${Color.blueLight};
+  }
+`
+let scaleWithBounce = keyframes`
+    0% {
+        transform: scaleY(0)
+    }
+    80% {
+        transform: scaleY(1.05)
+    }
+    100% {
+        transform: scaleY(1)
+    }
+`
+let SidebarAccountMenu = styled.div`
+  position: absolute;
+  left: 0;
+  right: 0;
+  background-color: ${Color.white};
+  color: ${Color.text};
+  border-radius: ${SizeUnit(0.25)};
+  margin-top: ${SizeUnit(0.25)};
+  margin-left: ${SizeUnit(0.15)};
+  margin-right: ${SizeUnit(0.15)};
+  z-index: ${ZIndex.SidebarMenu};
+  transform: scaleY(0);
+  opacity: 0;
+  transform-origin: top center;
+  transition: transform ${AnimDuration.short} ease,
+    opacity ${AnimDuration.short} linear;
+
+  &.isVisible {
+    transform: scaleY(1);
+    animation: ${scaleWithBounce} 300ms ease-in-out;
+    opacity: 1;
+  }
+`
+let SidebarAccountMenuHeader = styled.header`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 1px dotted ${Color.grayLight};
+  padding-top: ${SizeUnit(0.25)};
+  padding-bottom: ${SizeUnit(0.25)};
+  padding-left: ${SizeUnit(0.5)};
+  padding-right: ${SizeUnit(0.5)};
+`
+let SidebarAccountMenuLogo = styled(TiltCloudLogoSvg)`
+  fill: ${Color.text};
+`
+let SidebarAccountMenuLearn = styled.a`
+  font-family: ${Font.sansSerif};
+  font-size: ${FontSize.smallest};
+  color: ${Color.grayLight};
+`
+let SidebarAccountMenuContent = styled.div`
+  font-family: ${Font.monospace};
+  font-size: ${FontSize.default};
+  color: ${Color.grayLight};
+  padding: ${SizeUnit(0.5)};
+
+  p + p {
+    margin-top: ${SizeUnit(0.25)};
+  }
+
+  strong {
+    font-weight: bold;
+    color: ${Color.text};
+  }
+  small {
+    font-size: ${FontSize.small};
+  }
+  &.is-signedIn {
+    margin-top: ${SizeUnit(0.3)};
+    text-align: right;
+  }
+  &.is-signedOut {
+  }
+`
+let MenuContentTeamInTiltfile = styled.small`
+  opacity: 0;
+  margin-top: 0;
+  display: block;
+  transition: opacity ${AnimDuration.default} ease;
+  transition-delay: ${AnimDuration.short};
+`
+let MenuContentTeamName = styled.strong`
+  transition: background-color ${AnimDuration.default} ease;
+  transition-delay: ${AnimDuration.short};
+`
+let MenuContentTeam = styled.p`
+  &:hover {
```

Comment:
fwiw, i usually try to keep these together with the component, i.e.

let MenuContentTeam = styled.p``
let MenuContentTeamInTiltfile = styled.div`
  ${MenuContentTeam}:hover & {
    ...
  }
`
```

so that it's easier to see all the styling for a component in one place